charts/sck-otel/templates/_pod.tpl

```diff
@@ -94,6 +98,11 @@ containers:
         readOnly: true
       {{- end }}
 volumes:
+  {{- if .Values.journaldLogs.enabled }}
+  - name: journalpath
+    hostPath:
+      path: /run/log/journal
```

Comment:
The path shouldn't be hardcoded. it should take `.Values.journaldLogs.directory`

Reply:
Done